controllers/util/tests/tests.go

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright 2022 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"strings"
+
+	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
+	clustersyncer "github.com/IBM/ibm-block-csi-operator/controllers/syncer"
+	"github.com/IBM/ibm-block-csi-operator/pkg/config"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	nodeContainerName = clustersyncer.NodeContainerName
+	controllerContainerName = clustersyncer.ControllerContainerName
+	defaultControllerByName map[string]csiv1.IBMBlockCSIControllerSpec
+	defaultNodeByName map[string]csiv1.IBMBlockCSINodeSpec
+)
+var err = config.LoadDefaultsOfIBMBlockCSI()
```

Comment:
where is `err` being checked? consider calling `LoadDefaultsOfIBMBlockCSI` in `ibmblockcsi_controller_test.go`

Reply:
it doesn't being checked, I just wanted that we will run the function, and wanted that it will be in tests.go file that it can serve other tests that may use it

Reply:
if we ever need it in other tests, we could always find a common place, but I wouldn't count on it to happen.
regardless, I think the tests should fail with an indicative error if there is a problem loading the cr yaml content

Reply:
done